pkg/controller/daemon/daemon_controller.go

```diff
@@ -391,7 +391,7 @@ func (dsc *DaemonSetsController) addHistory(obj interface{}) {
 		if ds == nil {
 			return
 		}
-		klog.V(4).Infof("ControllerRevision %s added.", history.Name)
+		klog.V(4).InfoS("ControllerRevision added.", "controllerRevision", klog.KObj(history))
```

Comment:
Could you keep the format `Do not end the message with a period`?
and the rest also need to fix.

Reply:
Fixed, but FYI I did not add these periods.